oci/spec_opts.go

```diff
@@ -231,6 +231,16 @@ func WithProcessArgs(args ...string) SpecOpts {
 	}
 }
 
+// WithProcessCommandLine replaces the command line on the generated spec
+func WithProcessCommandLine(cmdLine string) SpecOpts {
```

Comment:
This only makes sense for Windows. I think we should set it in the oci_win file itself.